programmers/unfinished-runner.js

```diff
@@ -0,0 +1,22 @@
+const participant = ["leo","kiki","eden","leo"];
+const completion = ["eden", "kiki"];
+function solution(participant, completion) {
+    let  counts = {};
+    participant.forEach(function(i){
```

Comment:
띄어쓰기도 조금만 더 신경 써주세요~
```suggestion
    participant.forEach(function(i) {
```